src/pages/Pets.jsx

```diff
@@ -1,132 +1,278 @@
-                                <p className={`${styles.cardDesc} uk-margin-remove`}>Havenese</p>
-                            </div>
-                            <div>
-                                <img src={dog} alt="animal-icon" />
-                            </div>
-                        </div>
-                        <div className={`$ uk-flex uk-flex-between uk-flex-middle`}>
-                            <div className={`uk-margin-small-top`}>
-                            <Link to="/pet">
-                                    <button className={`${styles.details}`}>View Details</button>
-                                </Link>
-                            </div>
-                            <div>
-                                <div className={`uk-flex uk-flex-middle`}>
-                                    <span className="uk-margin-small-right"><GoLocation /></span>
-                                    <p className={`uk-margin-remove ${styles.locText}`}>Seattle, WA</p>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div className="uk-width-1-1@s uk-width-1-3@m uk-margin-small-top">
-                    <div class={`${styles.cardBorder} uk-card uk-card-small uk-card-default`}>
-                        <div>
-                            <img src={placeholder} className={`${styles.img}`} alt="dog" />
-                        </div>
-                        <div className={`uk-flex uk-flex-middle uk-flex-between`}>
-                            <div>
-                            <h4 className={`${styles.cardsub} uk-margin-small-top uk-margin-remove-bottom`}>Luna</h4>
-                                <p className={`${styles.cardDesc} uk-margin-remove`}>Havenese</p>
-                            </div>
-                            <div>
-                                <img src={dog} alt="animal-icon" />
-                            </div>
-                        </div>
-                        <div className={`$ uk-flex uk-flex-between uk-flex-middle`}>
-                            <div className={`uk-margin-small-top`}>
-                            <Link to="/pet">
-                                    <button className={`${styles.details}`}>View Details</button>
-                                </Link>
-                            </div>
-                            <div>
-                                <div className={`uk-flex uk-flex-middle`}>
-                                    <span className="uk-margin-small-right"><GoLocation /></span>
-                                    <p className={`uk-margin-remove ${styles.locText}`}>Seattle, WA</p>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+            <div>
```

Comment:
Refactor these codes; all of it. Make it shorter and more maintainable. You do not need these nested ternary operators to render the name of the `activeTab`. Simply doing 

`<h3 className={styles.header}>{activeTab}s</h3>` will serve.